modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc

```diff
@@ -7,7 +7,7 @@
 = Accessing third-party monitoring web service APIs
 
 [role="_abstract"]
-You can directly access third-party web service APIs from the command line for monitoring stack components such as Prometheus, Alertmanager, Thanos Ruler, and Thanos Querier. 
+You can directly access third-party web service APIs from the command line for the following monitoring stack components: Prometheus, Alertmanager, Thanos Ruler, and Thanos Querier. 
 
 The following example commands show how to query the service API receivers for Alertmanager.
 This example requires that the associated user account be bound against the `monitoring-alertmanager-edit` role in the `openshift-monitoring` namespace and that the account has the privilege to view the route.
```

Comment:
hmm `monitoring-alertmanager-edit` should allow access to all Alertmanager API endpoints. I'll test it on my side.

Reply:
Lmk what you find @simonpasquier. If the Alertmanager API is not available to users with the `monitoring-alertmanager-edit` role, then I think we should log a separate issue against this content and fix it in a separate PR. This fix would be out of scope for the current PR, which only intends to remove references to Grafana.

Reply:
please see the whole page, there is one Prerequisite
```
The following example commands show how to query the service API receivers for Alertmanager. This example requires that the associated user account be bound against the monitoring-alertmanager-edit role in the openshift-monitoring namespace and that the account has the privilege to view the route. This access only supports using a Bearer Token for authentication.
```
https://github.com/openshift/openshift-docs/pull/41384#discussion_r807687996 
mentioned the monitoring-alertmanager-edit role is not enough to see the alertmanager APIs, we changed to as it is now

Reply:
> please see the whole page, there is one Prerequisite
> 
> ```
> The following example commands show how to query the service API receivers for Alertmanager. This example requires that the associated user account be bound against the monitoring-alertmanager-edit role in the openshift-monitoring namespace and that the account has the privilege to view the route. This access only supports using a Bearer Token for authentication.
> ```
> 
> [#41384 (comment)](https://github.com/openshift/openshift-docs/pull/41384#discussion_r807687996) mentioned the monitoring-alertmanager-edit role is not enough to see the alertmanager APIs, we changed to as it is now

Even given the host, a user with `monitoring-alertmanager-edit` can't access alert receivers api

Reply:
having said that it shouldn't block this PR since it's an orthogonal issue to the change.

Reply:
I agree with @simonpasquier that this is an orthogonal issue outside the scope of this PR, and I don't think it should block this PR from being merged. @lihongyan1 - if you disagree, please state your reasons. But if you both agree that this isn't a blocker, I'll need you both to approve this PR as-is, so that I can get it merged. Then, one of us can write up a new docs Jira issue describing the problem with the Alertmanager sample provided and instructions on how to address the issue. Thanks!

Reply:
filed a bug to track the issue
https://bugzilla.redhat.com/show_bug.cgi?id=2102601

Reply:
Thanks all. Marking this as resolved.